docs/dependencies.md

```diff
@@ -58,13 +58,15 @@ type DependencyParamsIOST = {
   project?: string;
   podspecPath?: string;
   sharedLibraries?: string[];
+  buildTypes?: string[];
```

Comment:
I would keep the original naming to not introduce additional layer of confusion. We already have `scriptPhases` that are CocoaPods-specific (and others, such as `sourceDir`) and will most likely continue doing so in the future. What you think?